docs/deployment_with_telemetry.md

```diff
@@ -53,6 +53,7 @@ services:
         environment:
             - VERBOSITY=5
             - ETHEREUM_CHAIN_ID=1
+            - RPC_ADDR=0.0.0.0:60557
```

Comment:
```suggestion
            - RPC_ADDR=127.0.0.1:60557
```

Let's set this to `127.0.0.1` by default and add a comment about how to expose your API to the public or to your own servers if you want to.